static/js/recent_topics.js

```diff
@@ -32,6 +42,9 @@ exports.process_message = function (msg) {
     }
     topic_data.participated = is_ours || topic_data.participated;
     topic_data.muted = topic_data.muted || muting.is_topic_muted(msg.stream_id, msg.topic);
+    if (initialized) {
+        exports.inplace_rerender(key);
+    }
```

Comment:
This might end up being bad when processing 1K messages in a batch from the server.  I think what we probably want is to check the length of `messages`; if it's <5, we can do the inplace thing, otherwise, we should just do a full rerender.

Reply:
Done